websites/V/VGMdb/dist/metadata.json

```diff
@@ -10,18 +10,12 @@
 		"vi_VN": "VGMdb là dự án cộng đồng dành riêng cho việc lưu trữ âm nhạc và tranh ảnh của trò chơi điện tử."
 	},
 	"url": "vgmdb.net",
-	"version": "1.0.3",
+	"version": "1.0.4",
 	"logo": "https://i.imgur.com/mKSqsgJ.png",
 	"thumbnail": "https://i.imgur.com/n3L2Bom.png",
 	"color": "#B90000",
 	"category": "music",
-	"tags": [
-		"video",
-		"game",
-		"music",
-		"database",
-		"album"
-	],
+	"tags": ["video", "game", "music", "database", "album"],
```

Comment:
Revert this change, Prettier prefers the former version

Reply:
Not sure because this is actually Prettier's doing, but fixed reverted nonetheless